app/code/community/Bolt/Boltpay/Block/Checkout/Boltpay.php

```diff
@@ -557,6 +557,15 @@ function getConfigSelectors()
         return json_encode(array_filter(explode(',', Mage::getStoreConfig('payment/boltpay/selectors'))));
     }
 
+    /**
+     * Returns the Replace Button Selectors configuration.
+     * @return string
+     */
+    public function getMiniCartSelector()
+    {
+        return json_encode(array_filter(explode(',', Mage::getStoreConfig('payment/boltpay/minicart_selector'))));
```

Comment:
I don't see minicart_selector is defined. Is this used?

Reply:
The method uses in app/design/frontend/base/default/template/boltpay/replace.phtml::56 string